### ISSUE TITLE
Project Setup with Angular JS

### ISSUE THREAD

**Issue by MPSebastian** [amosproj/amos2023ss01-apache-pulsar-ui#9]
## User story
1. As a team
2. We want a well structured, modularzied project setup
3. So that our codebase stays organized.

## Acceptance criteria
* autogeneration of folder structure by the Angular CLI
* project can be build and run localy 
* setup linter for `ng lint` command

## Definition of done (DoD)
* Added only after week 5
* The same for all features
* Here goes the project specific part

## DoD general criteria
* Feature has been fully implemented
* Feature has been merged into the mainline
* All acceptance criteria were met
* Product owner approved features
* All tests are passing
* Developers agreed to release
